flapjack/config.py

```diff
@@ -9,6 +9,7 @@
     'Common': {
         'workdir': '~/flapjack',
         'checkoutdir': '${workdir}/checkout',
+        'shell_prompt': '(flapjack) [\\u@\h \W]\$$ ',
```

Comment:
What do you think about changing this to a prompt prefix? That way, if you have a complicated `PS1` already like I do, you don't need to duplicate it in the config file. Alternatively, introduce an extra escape code for including the outer environment's `PS1`.

Reply:
Absolutely, I really like the idea.